prover-stats-api/test/prover_task_test.go

```diff
@@ -116,10 +116,10 @@ func getResp(t *testing.T, url string) interface{} {
 	byt, err := io.ReadAll(resp.Body)
 	assert.NoError(t, err)
 
-	res := new(apitypes.Response)
+	res := new(types.Response)
 	assert.NoError(t, json.Unmarshal(byt, res))
 	t.Log("----byt is ", string(byt))
```

Comment:
This is redundant Log? Although don't involve to this PR.

Reply:
fixed [10977c8](https://github.com/scroll-tech/scroll/pull/938/commits/10977c806e2eb1465ef8b58da53af444049b252d)